content/en/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -471,6 +471,11 @@ other words, then kubelet may refuse to start Pods with errors similar to  "conf
 SELinux labels of volume", while there are no conflicting labels in Pods. Make sure
 nodes are [fully drained](/docs/tasks/administer-cluster/safely-drain-node/)
 before restarting kubelet.
+In Kubernetes 1.26 and newer, the kubelet reads SELinux options from the operating system
+and draining is no longer required before kubelet restart. Significant parts of the kubelet
+code was refactored, not only for SELinux, and resulted in a new feature gate
+`NewVolumeManagerReconstruction` introduced in Kubernetes 1.27 and enabled by default,
+to have an opt-out in case of a bug in the new implementation.
```

Comment:
IMO, a better way to document this is to delete **all** the contents of the note and replace it with:
```markdown
<!-- remove after Kubernetes v1.30 is released -->
If you are running Kubernetes v1.25, refer to the v1.25 version of this task page:
[Configure a Security Context for a Pod or Container](https://v1-25.docs.kubernetes.io/docs/tasks/configure-pod-container/security-context/) (v1.25).  
There is an important note in that documentation about a situation where the kubelet
can lose track of volume labels after restart.
```

Perhaps we should put a similar note on other pages, such as https://kubernetes.io/docs/tasks/debug/debug-cluster/ ?
In _that_ page, it does make sense to mention “conflicting SELinux labels of volume” etc. Imagine if you're troubleshooting a cluster, search the web for that error, and you see two search results:
1. Configure a Security Context for a Pod or Container
1. Troubleshooting Clusters

You're going to pick #<span/>2, I think. I would.

---

The place to describe the difference between changes is in the release announcement and the changelog; if we don't need to mention it in the docs, we shouldn't.

Reply:
In 1.25 SELinuxMountReadWriteOncePod is alpha and disabled by default, do we want to add troubleshooting for  those features?

Reply:
We want to help end users troubleshoot if they see an error “conflicting SELinux labels of volume”. I don't mind what the advice is, but I do think we should frame it as troubleshooting.

Reply:
Valid examples of advice:

- “You should upgrade if you can”
- “Try running v1.24 kubelets instead, this is a supported config”
- “Turn off SELinux and use appsecarmorcomp&trade;”

Reply:
Replaced the note as suggested, only in security-context.md for now.